recording-oracle/src/modules/campaigns/campaigns.repository.ts

```diff
@@ -31,14 +31,17 @@ export class CampaignsRepository extends Repository<CampaignEntity> {
     const timeAgo = dayjs().subtract(1, 'day').toDate();
 
     const results = await this.createQueryBuilder('campaign')
-      .where('campaign.status = :status', { status: CampaignStatus.ACTIVE })
-      .andWhere(
```

Comment:
Chaining these syntax sugar operators ends up in query w/o brackets and it's read as: `(campaign status = active AND endDate <= now) OR (....` due to SQL operator precedence.

In order to fix that we either have to use [TypeORM's brackets](https://typeorm.io/docs/query-builder/select-query-builder/#:~:text=You%20can%20add%20a%20complex%20WHERE%20expression%20into%20an%20existing%20WHERE%20using%20Brackets) which makes code less readable or just write a single `where` in a form that we need